frontend/components/PopularArticlesBlockList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Article from './Article'
+
+export default function PopularArticlesBlockList({ articles }) {
+    return (
+        <section className='flex justify-center'>
+            <div className='flex flex-col'>
+                <h1 className={`flex justify-center text-4xl fonts-title_popular-block`}>Популярное</h1>
```

Comment:
Зачем здесь фигурные кавычки + обратные кавычки (используемые для строковой интерполяции), когда возвращается статичная строка?

```suggestion
                <h1 className='flex justify-center text-4xl fonts-title_popular-block'>Популярное</h1>
```